ruby/config/database.yml

```diff
@@ -0,0 +1,6 @@
+development:
```

Comment:
It's not securely to add a file with credentials from the database in your git repository. Please, add this file to `.gitignore`

Reply:
Please, add this file to .gitignore, it is still in your branch